gbfs.md

```diff
@@ -173,6 +175,44 @@ email             | Optional  | A single contact email address for customers to
 timezone          | Yes       | The time zone where the system is located. Time zone names never contain the space character but may contain an underscore. Please refer to the "TZ" value in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for a list of valid values
 license_url       | Optional  | A fully qualified URL of a page that defines the license terms for the GBFS data for this system, as well as any other license terms the system would like to define (including the use of corporate trademarks, etc)
 
+### vehicle_types.json
+
+The following fields are all attributes within the main "data" object for this feed.
+
+Field Name        | Required  | Defines
+------------------| --------- | ----------
+vehicle_types          | Yes       | Array that contains one object per vehicle type in the system as defined below
+\- vehicle_type_id      | Yes       | Unique identifier of a vehicle type. See [Field Definitions](#field-definitions) above for ID field requirements
+\- form_factor      | Yes       | An enumerable describing the vehicle's general form factor. <br /><br />Current valid values are:<br /><ul><li>`bicycle`</li><li>`car`</li><li>`moped`</li><li>`other`</li><li>`scooter`</li></ul>
+\- propulsion_types | Yes | An array consisting of enumerables that describe the propulsion type of the vehicle. <br /><br />A device may have one or more values from the propulsion_type, depending on the number of modes of operation. For example, a scooter that can be powered by foot or by electric motor would have the propulsion_type represented by the array ['human', 'electric']. A bicycle with pedal-assist would have the propulsion_type represented by the array ['human', 'electric_assist'] if it can also be operated as a traditional bicycle. A car with an internal combustion engine would be represented by the array ['combustion']<br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was copied from [City of Los Angeles Mobility Data Specification](https://github.com/CityOfLosAngeles/mobility-data-specification/blob/73995a151f0a1d67aab3d617a4693f8f81967936/provider/README.md#propulsion-types)
+\- max_range_meters | Conditionally Required | If the vehicle has a motor (as indicated by having one or more values other than `human` in the `propulsion_types` array), this field is required. This represents the furthest distance in meters that the vehicle can travel without recharging or refueling when it has the maximum amount of energy potential (for example a full battery or full tank of gas).
+\- name | Optional | The public name of this vehicle type.
```

Comment:
@evansiroky I'm curious the rationale behind making the `name` optional. My initial thought was that this might complicate things for consumers who will then need to invent language around displaying this data, but as a consumer you probably have much better insights into whether this is actually a problem or not.

Reply:
As a consumer, I would prefer this field to be required. However, it seems that a number of systems don't have names for their vehicles. Take for example [Pacers Bikeshare](https://www.pacersbikeshare.org/how-it-works). I can't find any info on the models they call their bikes - even though they have both regular and e-bikes. So the thought was that maybe some producers don't call their bike models anything and maybe this field should be optional to reflect that. I'm open to making this required.